test-ts/utils.ts

```diff
@@ -758,3 +775,34 @@ export async function prepareOverflow(
     await voteTx.sendAndWaitForReceipt({ from: voter.address });
   }
 }
+
+export async function updateMaxNumberOfGroups(
+  accountAddress: string,
+  election: ElectionWrapper,
+  signerWithCelo: SignerWithAddress,
+  updateValue: boolean
+) {
+  const sendFundsTx = await signerWithCelo.sendTransaction({
+    value: parseUnits("1"),
+    to: accountAddress,
+  });
+  await sendFundsTx.wait();
+  await impersonateAccount(accountAddress);
+
+  const accountsContract = await hre.kit.contracts.getAccounts();
+  const createAccountTxObject = accountsContract.createAccount();
+  await createAccountTxObject.send({
+    from: accountAddress,
+  });
+  // TODO: once contractkit updated - use just election contract from contractkit
```

Comment:
Is there a PR out for this change? Could it be referenced here?

Reply:
I would prefer to not to put it into this PR